api/admin/controller/individual_admin_settings.py

```diff
@@ -83,6 +83,8 @@ def validate_form_fields(self, email):
         # eventually also check whether the input is formatted as a valid email address.
         if not email:
             return INCOMPLETE_CONFIGURATION
+        if not self.is_email(email):
+            return INVALID_EMAIL
```

Comment:
Since there are multiple email fields on the form, it would be useful to set a detail explaining which email address is bad. Something like this:

`return INVALID_EMAIL.detailed(_('"%(address)s" is not a valid email address", address=email))`